packages/strapi-plugin-upload/admin/src/components/CardPreview/index.js

```diff
@@ -4,19 +4,34 @@ import PropTypes from 'prop-types';
 import FileIcon from '../FileIcon';
 import Wrapper from './Wrapper';
 import Image from './Image';
+import BrokenFile from '../../icons/BrokenFile';
+
+const CardPreview = ({ hasError, url, type }) => {
+  const isFile = !type.includes('image') && !type.includes('video');
```

Comment:
I think you might need to create a util for this so you can use it in  other components, you can even improve it so it returns the type directly using a regex.